### PR TITLE
Dutch Translation for Galactricraft

### PR DESCRIPTION
**Pull request by Trico-Everfire**
My apologies if it's a little rough, I use English more than my native language of Dutch so I have trouble with  phrasing certain words and sentences (Dutch is a worse language than English honestly.)

### CODE REVIEW

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 1]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.detailed_iron_decoration_stairs": "Gedetailleerde Ijzere Decoratie Trappen",
+  "block.galacticraft.detailed_iron_decoration_slab": "Gedetailleerde Ijzere Decoratie Plaat",
+  "block.galacticraft.meteoric_iron_decoration": "Gewone Meteorische Ijzer Decoratie",
+  "block.galacticraft.meteoric_iron_decoration_wall": "Gewone Meteorische Ijzer Decoratie Muur",
+  "block.galacticraft.meteoric_iron_decoration_stairs": "Gewone Meteorische Ijzer Decoratie Trappen",
+  "block.galacticraft.meteoric_iron_decoration_slab": "Gewone Meteorische Ijzer Decoratie Plaat",
+  "block.galacticraft.detailed_meteoric_iron_decoration": "Gedetailleerde Meteorische Ijzer Decoratie",
+  "block.galacticraft.detailed_meteoric_iron_decoration_wall": "Gedetailleerde Meteorische Ijzer Decoratie Muur",
+  "block.galacticraft.detailed_meteoric_iron_decoration_stairs": "Gedetailleerde Meteorische Ijzer Decoratie Trappen",
+  "block.galacticraft.detailed_meteoric_iron_decoration_slab": "Gedetailleerde Meteorische Ijzer Decoratie Plaat",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:114]: Ijzere -> Ijzeren
Meteorische Ijzer -> Meteorische Ijzeren

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 2]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.cobbled_moon_rock_stairs": "geplaveide Maan Steen Trappen",
+  "block.galacticraft.cobbled_moon_rock_slab": "geplaveide Maan Steen Plaat",
+  "block.galacticraft.moon_basalt_brick": "Maan Basalt Bakstenen",
+  "block.galacticraft.moon_basalt_brick_wall": "Maan Basalt Bakstenen Muur",
+  "block.galacticraft.moon_basalt_brick_stairs": "Maan Basalt Bakstenen Trappen",
+  "block.galacticraft.moon_basalt_brick_slab": "Maan Basalt Bakstenen Plaat",
+  "block.galacticraft.cracked_moon_basalt_brick": "Cracked Maan Basalt Bakstenen",
+  "block.galacticraft.cracked_moon_basalt_brick_wall": "Cracked Maan Basalt Bakstenen Muur",
+  "block.galacticraft.cracked_moon_basalt_brick_stairs": "Cracked Maan Basalt Bakstenen Trappen",
+  "block.galacticraft.cracked_moon_basalt_brick_slab": "Cracked Maan Basalt Bakstenen Plaat",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:142]: Cracked -> Gebarsten

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 3]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.titanium_decoration_stairs": "Gewone Titanium Decoratie Trappen",
+  "block.galacticraft.titanium_decoration_slab": "Gewone Titanium Decoratie Plaat",
+  "block.galacticraft.detailed_titanium_decoration": "Gedetailleerde Titanium Decoratie",
+  "block.galacticraft.detailed_titanium_decoration_wall": "Gedetailleerde Titanium Decoratie Muur",
+  "block.galacticraft.detailed_titanium_decoration_stairs": "Gedetailleerde Titanium Decoratie Trappen",
+  "block.galacticraft.detailed_titanium_decoration_slab": "Gedetailleerde Titanium Decoratie Plaat",
+  "block.galacticraft.cobbled_moon_rock": "geplaveide Maan Steen",
+  "block.galacticraft.cobbled_moon_rock_wall": "geplaveide Maan Steen Muur",
+  "block.galacticraft.cobbled_moon_rock_stairs": "geplaveide Maan Steen Trappen",
+  "block.galacticraft.cobbled_moon_rock_slab": "geplaveide Maan Steen Plaat",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:134]: geplaveide -> Geplaveide

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 4]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.vacuum_glass": "Vacuum Glass",
+  "block.galacticraft.vacuum_glass_clear": "Doorzichtig Vacuüm Glas",
+  "block.galacticraft.vacuum_glass_strong": "Sterk Vacuüm Glas",
+  "block.galacticraft.coal_generator": "Kolen Generator",
+  "block.galacticraft.compressor": "Compressor",
+  "block.galacticraft.electric_compressor": "Elektrische Compressor",
+  "block.galacticraft.electric_furnace": "Elektrische Oven",
+  "block.galacticraft.electric_arc_furnace": "Elektrische Boogoven",
+  "block.galacticraft.circuit_fabricator": "Circuitbouwer",
+  "block.galacticraft.basic_solar_panel": "Basis ZonnePaneel",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:164]: ZonnePaneel -> Zonnepaneel

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 5]
```diff
@@ -0,0 +1,510 @@
...
+  "item.galacticraft.desh_shovel": "Desh schop",
+  "item.galacticraft.desh_pickaxe": "Desh Pikhouweel",
+  "item.galacticraft.desh_axe": "Desh Hakbijl",
+  "item.galacticraft.desh_hoe": "Desh Schoffel",
+  "item.galacticraft.titanium_sword": "Titanium Zwaard",
+  "item.galacticraft.titanium_shovel": "Titanium schop",
+  "item.galacticraft.titanium_pickaxe": "Titanium Pikhouweel",
+  "item.galacticraft.titanium_axe": "Titanium Hakbijl",
+  "item.galacticraft.titanium_hoe": "Titanium Schoffel",
+  "item.galacticraft.meteoric_iron_ingot": "meteorisch Ijzerstaaf",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:212]: meteorisch -> Meteorische

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 6]
```diff
@@ -0,0 +1,510 @@
...
+  "item.galacticraft.heavy_duty_pickaxe": "Op Zwaar Werk Berekende Pikhouweel",
+  "item.galacticraft.heavy_duty_axe": "Op Zwaar Werk Berekende Hakbijl",
+  "item.galacticraft.heavy_duty_hoe": "Op Zwaar Werk Berekende Schoffel",
+  "item.galacticraft.desh_sword": "Desh Zwaard",
+  "item.galacticraft.desh_shovel": "Desh schop",
+  "item.galacticraft.desh_pickaxe": "Desh Pikhouweel",
+  "item.galacticraft.desh_axe": "Desh Hakbijl",
+  "item.galacticraft.desh_hoe": "Desh Schoffel",
+  "item.galacticraft.titanium_sword": "Titanium Zwaard",
+  "item.galacticraft.titanium_shovel": "Titanium schop",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:208]: schop -> Schop

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 7]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.oxygen_bubble_distributor": "Bellenverdeler",
+  "block.galacticraft.lunar_cartography_table": "Maan Cartografie Tafel",
+  "item.galacticraft.sensor_glasses": "Sensor Bril",
+  "item.galacticraft.heavy_duty_sword": "Op Zwaar Werk Berekende Zwaard",
+  "item.galacticraft.heavy_duty_shovel": "Op Zwaar Werk Berekende schop",
+  "item.galacticraft.heavy_duty_pickaxe": "Op Zwaar Werk Berekende Pikhouweel",
+  "item.galacticraft.heavy_duty_axe": "Op Zwaar Werk Berekende Hakbijl",
+  "item.galacticraft.heavy_duty_hoe": "Op Zwaar Werk Berekende Schoffel",
+  "item.galacticraft.desh_sword": "Desh Zwaard",
+  "item.galacticraft.desh_shovel": "Desh schop",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:203]: schop -> Schop

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 8]
```diff
@@ -0,0 +1,510 @@
...
+  "item.galacticraft.titanium_sword": "Titanium Zwaard",
+  "item.galacticraft.titanium_shovel": "Titanium schop",
+  "item.galacticraft.titanium_pickaxe": "Titanium Pikhouweel",
+  "item.galacticraft.titanium_axe": "Titanium Hakbijl",
+  "item.galacticraft.titanium_hoe": "Titanium Schoffel",
+  "item.galacticraft.meteoric_iron_ingot": "meteorisch Ijzerstaaf",
+  "item.galacticraft.titanium_ingot": "Titaniumstaaf",
+  "item.galacticraft.desh_ingot": "Deshstaaf",
+  "item.galacticraft.lead_ingot": "loodstaaf",
+  "item.galacticraft.raw_meteoric_iron": "Rauw meteorisch Ijzer",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:216]: meteorisch -> Meteorisch

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 9]
```diff
@@ -0,0 +1,510 @@
...
+  "item.galacticraft.compressed_bronze": "Compressed Bronze",
+  "item.galacticraft.compressed_copper": "Compressed Koper",
+  "item.galacticraft.compressed_iron": "Compressed Ijzer",
+  "item.galacticraft.compressed_meteoric_iron": "Compressed meteorisch Ijzer",
+  "item.galacticraft.compressed_steel": "Compressed Staal",
+  "item.galacticraft.compressed_desh": "Compressed Desh",
+  "item.galacticraft.compressed_titanium": "Compressed Titanium",
+  "item.galacticraft.raw_desh": "Ongeraffineerd Desh",
+  "item.galacticraft.desh_stick": "Desh Stok",
+  "item.galacticraft.tin_canister": "Tinne Busje",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:237]: Tinne -> Tinnen

OR

Tinne -> Blikken

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 10]
```diff
@@ -0,0 +1,510 @@
...
+  "item.galacticraft.copper_canister": "Koperen Busje",
+  "item.galacticraft.carbon_fragments": "Koolstof Fragmenten",
+  "item.galacticraft.titanium_shard": "Titanium Scherf",
+  "item.galacticraft.iron_shard": "Ijzer Scherf",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:241]: Ijzer -> Ijzeren

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 11]
```diff
@@ -0,0 +1,510 @@
...
+  "item.galacticraft.titanium_dust": "Titanium Stof",
+  "item.galacticraft.solar_dust": "Zonne Stof",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:243]: Zonne Stof -> Zonnestof

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 12]
```diff
@@ -0,0 +1,510 @@
...
+  "item.galacticraft.moon_villager_spawn_egg": "Maan Dorpeling Spawn Ei",
+  "item.galacticraft.evolved_zombie_spawn_egg": "Geëvolueerd Zombie Spawn Ei",
+  "item.galacticraft.evolved_pillager_spawn_egg": "Geëvolueerd Plunderaar Spawn Ei",
+  "item.galacticraft.evolved_vindicator_spawn_egg": "Geëvolueerd Vindicator Spawn Ei",
+  "item.galacticraft.evolved_evoker_spawn_egg": "Geëvolueerd Evoker Spawn Ei",
+  "item.galacticraft.evolved_skeleton_spawn_egg": "Geëvolueerd Skelet Spawn Ei",
+  "item.galacticraft.evolved_creeper_spawn_egg": "Geëvolueerd Creeper Spawn Ei",
+  "item.galacticraft.evolved_spider_spawn_egg": "Geëvolueerd Spin Spawn Ei",
+  "item.galacticraft.throwable_meteor_chunk": "Gooibare Meteor Brok",
+  "item.galacticraft.hot_throwable_meteor_chunk": "Hete Gooibare Meteor Brok",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:329]: Gooibare -> Werpbare

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 13]
```diff
@@ -0,0 +1,510 @@
...
+  "tooltip.galacticraft.cheese_block": "Kaas Bloks zijn gemaakt van de kazen van de Maan, plaatsbaar en eetbaar.",
+  "tooltip.galacticraft.energy_storage_module": "De energie opslag module wordt gebruikt om grote hoeveelheden energie op te slaan voor later gebruik.",
+  "tooltip.galacticraft.energy_storage_cluster": "Energie Opslag Cluster wordt gebruikt om grote hoeveelheden energie op te slaan voor later gebruik. Slaat en geeft meer energie af dan zijn voorganger.",
+  "tooltip.galacticraft.glowstone_torch": "Gloeisteen fakkels kunnen het beste worden gebruikt om gebieden te verlichten waar er geen zuurstof is om houtfakkels te laten branden.",
+  "tooltip.galacticraft.glowstone_lantern": "Gloeisteen Lantaarns worden het best gebruikt om gebieden te verlichten waar er geen zuurstof is om lantaarns te laten branden.",
+  "tooltip.galacticraft.standard_wrench": "De meeste Galacticraft-machines kunnen worden gedraaid door met de rechtermuisknop te klikken met de moersleutel.",
+  "tooltip.galacticraft.oxygen_remaining": "Resterende Zuurstof: %s",
+  "tooltip.galacticraft.energy_remaining": "Resterende Energy: %s",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:352]: Energy -> Energie

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 14]
```diff
@@ -0,0 +1,510 @@
...
+  "tooltip.galacticraft.infinite": "Oneindig",
+  "tooltip.galacticraft.creative_only": "Alleen voor creatief",
+  "death.attack.gc.suffocation": "%s stierf vanwegen zuurstofgebrek",
+  "ui.galacticraft.alpha_warning.header": "WAARSCHUWING",
+  "ui.galacticraft.alpha_warning.content1": "Galacticraft is op did moment in ALPHA.",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:357]: did -> dit

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 15]
```diff
@@ -0,0 +1,510 @@
...
+  "ui.galacticraft.machine.current_energy": "Energie: %s",
+  "ui.galacticraft.machine.max_energy": "Maximale Energie: %s",
+  "ui.galacticraft.machine.current_oxygen": "Zuurstof: %s",
+  "ui.galacticraft.machine.max_oxygen": "Maximale Zuurstof: %s",
+  "ui.galacticraft.machine.gj_per_t": "%sGj/t",
+  "ui.galacticraft.machine.blocked_panels": "Zichtbaarheid: %s/9",
+  "ui.galacticraft.machine.collecting": "Verzamelen: %s/s",
+  "ui.galacticraft.machine.status.not_enough_leaves": "Niet genoeg bladeren",
+  "ui.galacticraft.machine.status.active": "Actief",
+  "ui.galacticraft.machine.status.idle": "Nietsdoend",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:373]: Ijdel would be more accurate

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 16]
```diff
@@ -0,0 +1,510 @@
...
+  "ui.galacticraft.machine.status.warming": "Waarschuwing",
+  "ui.galacticraft.machine.status.inactive": "inactief",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:375]: inactief -> Inactief

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 17]
```diff
@@ -0,0 +1,510 @@
...
+  "ui.galacticraft.bubble_distributor.visible": "Bubbel zichtbaar",
+  "ui.galacticraft.bubble_distributor.not_visible": "Bubbel onzichtbaar",
+  "ui.galacticraft.bubble_distributor.size": "Doel Grote:",
+  "ui.galacticraft.machine.fluid_inv.fluid": "Vloeistof: ",
+  "ui.galacticraft.machine.fluid_inv.amount": "Hoeveelheid: ",
+  "ui.galacticraft.machine.fluid_inv.empty": "Leeg",
+  "category.rei.circuit_fabricator": "Circuit Fabriceren",
+  "category.rei.compressing": "Comprimeren",
+  "galacticraft:moon": "Maan",
+  "commands.galacticraft.dimensiontp.success.multiple": "Geteleporteerd %s entiteiten Naar %s",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:458]: %s entiteiten geleteporteerd naar %s

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 18]
```diff
@@ -0,0 +1,510 @@
...
+  "commands.galacticraft.dimensiontp.success.single": "Geteleporteerd Naar %s",
+  "commands.galacticraft.dimensiontp.success.pos": "Geteleporteerd naar %s bij (%d, %d, %d)",
+  "commands.galacticraft.dimensiontp.failure.dimension": "Kan dimentie niet vinden",
+  "commands.galacticraft.dimensiontp.failure.already_in_dimension": "Al in dimentie %s",
+  "commands.galacticraft.dimensiontp.failure.entity": "Ongeldig entiteit",
+  "commands.galacticraft.gchouston.error": "We kunnen je signaal niet vinden!",
+  "commands.galacticraft.gchouston.cannot_detect_signal": "We kunnen je signaal niet vinden! Weet je zeker dat je in de ruimte zit?",
+  "commands.galacticraft.gchouston.confirm": "Er, Houston, we hebben een probleem... (Gebruik deze command nogmaals om te teleporteren.)",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:466]: deze command -> deze opdracht (more consistent with vanilla)

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 19]
```diff
@@ -0,0 +1,510 @@
...
+  "commands.galacticraft.gchouston.on_earth_already": "Ik hoef niet gered te worden!",
+  "commands.galacticraft.gchouston.success": "Je bent gered. Hopelijk de volgende keer meer geluk...",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:468]: de volgende keer meer geluk -> volgende keer beter

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 20]
```diff
@@ -0,0 +1,510 @@
...
+  "advancements.galacticraft.travel_moon.title": "Naar de Maan!",
+  "advancements.galacticraft.travel_moon.description": "Neem een Niveau 1 Raket Naar de Maan!",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:470]: Naar -> naar

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 21]
```diff
@@ -0,0 +1,510 @@
...
+  "config.galacticraft.energy.machines.oxygen_decompressor_energy_consumption_rate": "Energieverbruik van zuurstof decompressor/t",
+  "config.galacticraft.energy.machines.refinery_energy_consumption_rate": "Energieverbruik van raffinaderij/t",
+  "config.galacticraft.energy.machines.electric_furnace_energy_consumption_rate": "Energieverbruik van elektrische oven/t",
+  "config.galacticraft.energy.machines.energy_storage_module_storage_size": "Energie opslag grote van energie opslag module",
+  "config.galacticraft.energy.machines.energy_storage_size": "Standaard machine energie opslag grote",
+
+  "item.galacticraft.legacy_music_disc_mars": "Muziekschijf",
+  "item.galacticraft.legacy_music_disc_mimas": "Muziekschijf",
+  "item.galacticraft.legacy_music_disc_orbit": "Muziekschijf",
+  "item.galacticraft.legacy_music_disc_spacerace": "Muziekschijf",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:505]: Muziekschijf -> Muziekplaat (vanilla)

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 22]
```diff
@@ -0,0 +1,510 @@
...
+  "config.galacticraft.energy.machines.solar_panel_energy_production_rate": "Energie productie snelheid van zonne panelen/t",
+  "config.galacticraft.energy.machines.circuit_fabricator_energy_consumption_rate": "Energieverbruik van circuitfabrikant/t",
+  "config.galacticraft.energy.machines.electric_compressor_energy_consumption_rate": "Energieverbruik van elektrische compressor/t",
+  "config.galacticraft.energy.machines.oxygen_collector_energy_consumption_rate": "Energieverbruik van zuurstof verzamelaar/t",
+  "config.galacticraft.energy.machines.oxygen_compressor_energy_consumption_rate": "Energieverbruik van zuurstof compressor/t",
+  "config.galacticraft.energy.machines.oxygen_decompressor_energy_consumption_rate": "Energieverbruik van zuurstof decompressor/t",
+  "config.galacticraft.energy.machines.refinery_energy_consumption_rate": "Energieverbruik van raffinaderij/t",
+  "config.galacticraft.energy.machines.electric_furnace_energy_consumption_rate": "Energieverbruik van elektrische oven/t",
+  "config.galacticraft.energy.machines.energy_storage_module_storage_size": "Energie opslag grote van energie opslag module",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:499]: grote -> grootte

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 23]
```diff
@@ -0,0 +1,510 @@
...
+  "item.galacticraft.burger_bun": "Burger Bol",
+  "item.galacticraft.beef_patty": "Gekookte Rundvleespasteitje",
+  "item.galacticraft.ground_beef": "Rauw Rundvleespasteitje",
+  "item.galacticraft.oxygen_mask": "Zuurstof Masker",
+  "item.galacticraft.oxygen_gear": "Zuurstof Uitrusting",
+  "item.galacticraft.small_oxygen_tank": "Kleine Zuurstof Reservoir",
+  "item.galacticraft.medium_oxygen_tank": "Medium Zuurstof Reservoir",
+  "item.galacticraft.large_oxygen_tank": "Grote Zuurstof Reservoir",
+  "item.galacticraft.infinite_oxygen_tank": "Oneindige Zuurstof Reservoir",
+  "item.galacticraft.shield_controller": "schild Controller",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:304]: schild -> Schild

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 24]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.oxygen_storage_module": "Zuurstof Opslag Module",
+  "block.galacticraft.refinery": "Raffinaderij",
+  "block.galacticraft.aluminum_wire": "Aluminium Kabel",
+  "block.galacticraft.sealable_aluminum_wire": "Verzegelbare Aluminium Kabel",
+  "block.galacticraft.fluid_pipe": "Vloeistof Pijp",
+  "block.galacticraft.glass_fluid_pipe": "Glazen Vloeistof Pijp",
+  "block.galacticraft.square_light_panel": "Licht Paneel (Vierkant)",
+  "block.galacticraft.spotlight_light_panel": "Licht Paneel (Spotlicht)",
+  "block.galacticraft.linear_light_panel": "Licht Paneel (Lineair)",
+  "block.galacticraft.dashed_light_panel": "Licht Paneel (gestreept)",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:180]: gestreept -> Gestreept

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 25]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.meteoric_iron_block": "meteorisch Ijzer Blok",
+  "block.galacticraft.tin_ladder": "Tin Ladder",
+  "block.galacticraft.tin_decoration": "Gewone Tinne Decoratie",
+  "block.galacticraft.tin_decoration_wall": "Gewone Tinne Decoratie Muur",
+  "block.galacticraft.tin_decoration_stairs": "Gewone Tinne Decoratie Trappen",
+  "block.galacticraft.tin_decoration_slab": "Gewone Tinne Decoratie Plaat",
+  "block.galacticraft.detailed_tin_decoration": "Gedetailleerde Tinne Decoratie",
+  "block.galacticraft.detailed_tin_decoration_wall": "Gedetailleerde Tinne Decoratie Muur",
+  "block.galacticraft.detailed_tin_decoration_stairs": "Gedetailleerde Tinne Decoratie Trappen",
+  "block.galacticraft.detailed_tin_decoration_slab": "Gedetailleerde Tinne Decoratie Plaat",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:74]: Tinne -> Blikken

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 26]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.moon_basalt": "Maan Basalt",
+  "block.galacticraft.moon_basalt_bricks": "Maan Basalt Stenen",
+  "block.galacticraft.moon_basalt_slab": "Maan Basalt Plaat",
+  "block.galacticraft.moon_basalt_stairs": "Maan Basalt Trappen",
+  "block.galacticraft.moon_basalt_wall": "Maan Basalt Muur",
+  "block.galacticraft.moon_rock_stairs": "Maan Steen Trappen",
+  "block.galacticraft.moon_rock_slab": "Maan Steen Plaat",
+  "block.galacticraft.moon_rock_wall": "Maan Steen Muur",
+  "block.galacticraft.moon_dirt": "Maan Aarde",
+  "block.galacticraft.moon_copper_ore": "Maan Koper Erts",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:25]: Koper Erts -> Kopererts

File: src/main/resources/assets/galacticraft/lang/nl_nl.json [hunk 27]
```diff
@@ -0,0 +1,510 @@
...
+  "block.galacticraft.moon_tin_ore": "Maan Tin Erts",
```
Comment (martmists-gh) [inline src/main/resources/assets/galacticraft/lang/nl_nl.json:26]: Tin Erts -> Tinerts

**Review by martmists-gh — CHANGES_REQUESTED** [review]
Mostly capitalization, also some fixed translations.

**Comment by Trico-Everfire** [comment]
Yo, thanks for fixing it up, I am ashamed to admit that although it's my birth language, I haven't had a proper dutch conversation in text form since 2016, so some things just yeet over my head when translating, thanks for catching them mistakes.

**Comment by martmists-gh** [comment]
No problem, that's what reviews are for :) It doesn't help that a lot of this stuff is really awkward to say in Dutch either way.

**Pull request closed without merging**